Orange/canvas/report/report.py

```diff
@@ -216,16 +219,18 @@ def data(role=Qt.DisplayRole,
                     if row is None or col is None:
                         return model.headerData(col if row is None else row,
                                                 orientation, role)
-                    return model.data(model.index(row, col), role)
+                    return try_fmtnum(model.data(model.index(row, col), role))
```

Comment:
Do not reformat the number here, at least not like this.

Orange used to keep track of the number of decimal places it should print for a specific variable. For instance, the "ST by exercise" has only a single decimal in the heart_disease.tab file, and the "age" and the "number of vessels colored" have none, obviously. Orange used to record this when loading the data. Now all numeric values have three decimals.

I consider this a bug (#1581). After this is fixed, `model.data` that is used in, say, the Table widget, will (again) show the numeric values with appropriate number of decimals, so this function should not reformat it.